Client/Frontend/Browser/URLBarView.swift

```diff
@@ -223,11 +238,23 @@ class URLBarView: UIView {
             make.size.equalTo(URLBarViewUX.TabsButtonHeight)
         }
 
+        tabsButton.snp_makeConstraints { make in
+            make.centerY.equalTo(self.locationContainer)
+            make.trailing.equalTo(self)
+            make.width.height.equalTo(UIConstants.ToolbarHeight)
```

Comment:
Didn't make these constraints (just moved them from where they were), but that sounds OK to me.